Kinetic_space_partition/include/CGAL/KSP_3/Intersection_graph.h

```diff
@@ -64,8 +64,18 @@ class Intersection_graph {
     std::map<std::size_t, Kinetic_interval> intervals; // Maps support plane index to the kinetic interval. std::pair<FT, FT> is the barycentric coordinate and intersection time.
     Edge_property() : line(std::size_t(-1)), order(edge_counter++) { }
 
+    Edge_property(const Edge_property& other) {
+      line = other.line;
+      order = other.order;
+      faces = other.faces;
+      planes = other.planes;
+      crossed = other.crossed;
+      intervals = other.intervals;
+    }
+
     const Edge_property& operator=(const Edge_property& other) {
       line = other.line;
+      order = other.order;
```

Comment:
It is this commit, and in particular line 78 which leads to the timeouts (hanging)  in  [Ic-209](https://cgal.geometryfactory.com/CGAL/testsuite/results-6.0-Ic-209.shtml#Kinetic_space_partition).

As this fix seems to be the right one (a copy constructor in general copies all data members), we have to find out if not copying order is intentional and correct, or if the error is somewhere else.